squad_client/core/models.py

```diff
@@ -232,12 +232,44 @@ class TestJob(SquadObject):
 
 class TestRun(SquadObject):
 
+    def __init__(self):
+        self.__total_pass__ = 0
+        self.__total_fail__ = 0
+        self.__total_skip__ = 0
+        self.__total_xfail__ = 0
+        self.test_suites = []
+        self.metric_suites = []
+
     endpoint = '/api/testruns/'
     attrs = ['url', 'id', 'metadata_file', 'log_file',
              'created_at', 'completed', 'datetime', 'build_url',
              'job_id', 'job_status', 'job_url', 'resubmit_url', 'data_processed',
              'status_recorded', 'build', 'environment']
 
+    @property
+    def total_pass(self):
```

Comment:
As @mwasilew said yesterday those are expensive especially when we get to Android tests and ideally should be readily available on Testrun Obj itself on Squad. I will look into adding them there and then we modify the client once again to use them.